app/keepers/keepers.go

```diff
@@ -299,14 +291,58 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 		appKeepers.IBCKeeper.ChannelKeeper,
 	)
 
+	composablePrefix := sdk.GetConfig().GetBech32AccountAddrPrefix()
+
+	wasmDir := filepath.Join(homePath, "wasm")
+	wasmConfig, err := wasm.ReadWasmConfig(appOpts)
+	if err != nil {
+		panic(fmt.Sprintf("error while reading wasm config: %s", err))
+	}
+
+	// increase default wasm size in all wasmd related codes (as on Neutorn/Osmosis)
+	wasmtypes.MaxWasmSize *= 2
```

Comment:
can this function be called multiple times, and every time it will double the wasm size?
Why not hardcode/gov prop param?

Reply:
> gov prop param

cannot by design of wasmd code - it is just var.

> can this function be called multiple times, and every time it will double the wasm size?

go is shit. all is global mutable vars. anybody can call anything anywhere. in this case - app constructor app.New called only once - singleton. so I do not worry.  

doubling default for me is good. if we randomly add code which doubles without review - good luck such chain to live. and because we do not - nobody will double.